### ISSUE TITLE
Multisite to External Sites

### ISSUE THREAD

**Issue by Kameleon-Dean** [10up/distributor#484]
I have a multi site setup 5 sites where the distributor plugin is used to push from all the child sites into a main website just for standard posts, which works fine. Since using the plugin I have then needed to break apart the child sites into their own sites and remove the multi site functionality completely across the board but maintain the push functionality.

I can see the post meta records need to be manually amended which is fine for the mappings. I have setup an external connection on one of the old child sites to the master site, all authenticated etc. I can also pull from the master's posts. The problem is I can't push from the edit post screen of a child site.

The drop down window for the distributor plugin shows blank on an edit post page when trying to bring up the connections listing to create the map. 

![Annotation 2019-11-08 125847](https://user-images.githubusercontent.com/57530317/68478383-9d0bd580-0227-11ea-982f-265290a81e24.png)

Upon closer inspection using the inspector I can see there is some text wrapped in a display none with the message "There was an issue loading connections.".

No PHP or JavaScript errors. It can't determine any connections at this point given that an external exists and is fully connected.

**Environment information**
 - Device: HP Z2 Workstation
 - OS: Windows 10
 - Browser and version: Chrome: 78.0.3904.87
 - Distributor version: 1.5.0 (both sites)

Thanks,
Dean

**Comment by jeffpaul** [comment]
@Kameleon-Dean can you confirm that the post you were trying to push was published and not in draft mode?  Can you also confirm the status for the external connection for your master site on that child site as well?

**Comment by Kameleon-Dean** [comment]
Yes the post is published and the connection is fully authorised via the REST API using the application password plugin as recommended. No issues brought up when testing the external connection. I have also tried publishing a new post and the same issue occurs.

**Comment by jeffpaul** [comment]
@Kameleon-Dean ok, I'll try to replicate the multisite split to separate instances to see if that process has something weird going on.  If your troubleshooting finds anything else, please let me know... thanks!

**Comment by Kameleon-Dean** [comment]
@jeffpaul how did you get on? any updates on this?

**Comment by jeffpaul** [comment]
@Kameleon-Dean I haven't been able to fully test/replicate this yet, no.  It's still on my list, I'll get back to you once I've been able to fully test/replicate, thanks!

**Comment by Jimmi08** [comment]
Hi, guys, I am only testing this plugin and I hit on this issue because I have a similar problem.

I have test multisite and everything works.
Then for one child site, I have a single installation (not moved, created a new one)
After some struggling, I got verified external connection (green dot).
I can see listed that external site above edited post, I select it, push it and nothing happens. The post is not on another site and no  View option is there.

EDIT: the same behavior single site to a single site, so it will not be related that original site was multisite.

**Comment by jeffpaul** [comment]
@Jimmi08 can you confirm that you're following the [External Connections set up instructions](https://github.com/10up/distributor/#setup-external-connections-using-application-passwords)?  If so, then we'll need more information on your installs to better triage your situation, so providing Site Health details would be most helpful.

**Comment by Jimmi08** [comment]
@jeffpaul Thank you for your reply. we decided for multisite version for now. Sorry.  But I suppose that problem was on our side.
